ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ReportJobs.tsx

```diff
@@ -21,60 +22,75 @@ import { ensureBoolean, ensureStringArray } from 'utils/ensure';
 import useURLStringUnion from 'hooks/useURLStringUnion';
 import { RunState } from 'types/reportJob';
 import useAnalytics from 'hooks/useAnalytics';
+import useRestQuery from 'hooks/useRestQuery';
+import useURLSort from 'hooks/useURLSort';
+import { getRequestQueryStringForSearchFilter } from 'utils/searchUtils';
+import { getTableUIState } from 'utils/getTableUIState';
 import ConfigDetails from './ConfigDetails';
 import ReportRunStatesFilter, { ensureReportRunStates } from './ReportRunStatesFilter';
 import MyJobsFilter from './MyJobsFilter';
 
-function createMockData(scanConfig: ComplianceScanConfigurationStatus) {
-    const snapshots: ComplianceReportSnapshot[] = [
-        {
-            reportJobId: 'ab1c03ae-9707-43d1-932d-f948afb67b53',
-            scanConfigId: scanConfig.id,
-            name: scanConfig.scanName,
-            description: scanConfig.scanConfig.description,
-            reportStatus: {
-                completedAt: '2024-08-27T00:01:40.569402380Z',
-                errorMsg:
-                    "Error sending email notifications:  error: Error sending email for notifier 'fc99e179-57c1-4ba2-8e59-45dbf184c78c': Connection failed",
-                reportNotificationMethod: 'EMAIL',
-                reportRequestType: 'SCHEDULED',
-                runState: 'FAILURE',
-            },
-            reportData: scanConfig,
-            user: {
-                id: 'sso:3e30efee-45f0-49d3-aec1-2861fcb3faf6:c02da449-f1c9-4302-afc7-3cbf450f2e0c',
-                name: 'Test User',
-            },
-            isDownloadAvailable: false,
-        },
-    ];
-    return snapshots;
-}
-
 function getJobId(snapshot: ComplianceReportSnapshot) {
-    return snapshot.scanConfigId;
+    return snapshot.reportJobId;
 }
 
 function getConfigName(snapshot: ComplianceReportSnapshot) {
     return snapshot.name;
 }
 
+const sortOptions = {
+    sortFields: ['Compliance Report Completed Time'],
+    defaultSortOption: { field: 'Compliance Report Completed Time', direction: 'desc' } as const,
```

Comment:
Is `as const` for object literal in `defaultSortOption` or for `sortOptions` variable?

Reply:
For the `defaultSortOption`. I copypasted this from VM Reports. It seems like we do it there, too, but I couldn't find exactly why. 

We get this error when we remove it

```
Type 'string' is not assignable to type 'NonNullable<"desc" | "asc">'
```